frontend/cypress/integration/common/workload_logs.ts

```diff
@@ -0,0 +1,25 @@
+import { Given, Then, When } from "cypress-cucumber-preprocessor/steps";
+
+Given("I am on the {string} workload detail page", (workload) => {
+    // Go to workloads list
+    cy.get("#Workloads").click();
+
+    // Select "bookinfo" namespace
+    cy.get("#namespace-selector").click();
+    cy.get("#bulk-select-id").uncheck();
+    cy.get('#namespace-list-item\\[bookinfo\\] input').check();
```

Comment:
Perhaps better to pass the namespace as a parameter instead to hard-code the "bookinfo" one?